src/d3d12/D3D12_Functions.cpp

```diff
@@ -466,7 +464,7 @@ bool D3D12::InitializeImGui(size_t aBuffersCounts)
 
     ReloadFonts();
 
-    if (!ImGui_ImplDX12_CreateDeviceObjects(m_pCommandQueue.Get()))
+    if (!ImGui_ImplDX12_CreateDeviceObjects())
```

Comment:
as said on discord, we need to pass in command queue...

Reply:
I'll add this back. What is this doing?

Reply:
helps with hdr, crashing when alt+tabing etc (framebuffer resizes) a bit (not fully though), better reshade compatibility as we do not try to make bastardized command queue depending on who overrides dx functions, better windows 7 compat as either downlevel or normal command queue is used

a lot simply put tiny things together this was required for